src/mscorlib/src/System/Threading/ThreadPool.cs

```diff
@@ -129,10 +129,34 @@ internal void Remove(T e)
             }
         }
 
+        // Enusre full cache line occupied so no false sharing with other work items
+        [StructLayout(LayoutKind.Explicit, Size = 64)]
+        internal struct PaddedWorkItem
+        {
+            [FieldOffset(0)]
+            public IThreadPoolWorkItem Item;
+        }
+
+        // Enusre full cache line occupied so no false sharing with other work items
+        [StructLayout(LayoutKind.Explicit, Size = 64)]
+        internal struct PaddedQueueSegment
+        {
+            [FieldOffset(0)]
+            public volatile QueueSegment Segment;
+        }
+
+        // Enusre full cache line occupied so no false sharing with other indicies
```

Comment:
Enusre => Ensure in all of these cases

Technically this isn't ensuring it, either, as there are systems with cache line sizes greater than 64 bytes.  But they're less common.